### ISSUE TITLE
EventException - Could not pass event PlayerInteractEvent

### ISSUE THREAD

**Issue by mibby** [RockinChaos/ItemJoin#87]
ItemJoin 3.9.1

```
[16:11:05] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to ItemJoin v3.9.1
org.bukkit.event.EventException
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:46) ~[patched_1.11.2.jar:git-Paper-1059]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:233) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:200) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:196) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:946) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:26) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:5) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.11.2.jar:git-Paper-1059]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_121]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_121]
	at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:830) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:403) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:754) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:652) [patched_1.11.2.jar:git-Paper-1059]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: java.lang.IllegalStateException: Start block missed in BlockIterator
	at org.bukkit.util.BlockIterator.<init>(BlockIterator.java:170) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.util.BlockIterator.<init>(BlockIterator.java:233) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.util.BlockIterator.<init>(BlockIterator.java:268) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.entity.CraftLivingEntity.getLineOfSight(CraftLivingEntity.java:178) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.entity.CraftLivingEntity.getLineOfSight(CraftLivingEntity.java:204) ~[patched_1.11.2.jar:git-Paper-1059]
	at me.RockinChaos.itemjoin.Listeners.InteractCmds.getNearestEntityInSight(InteractCmds.java:64) ~[?:?]
	at me.RockinChaos.itemjoin.Listeners.InteractCmds.onInteractCmds(InteractCmds.java:50) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor129.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:44) ~[patched_1.11.2.jar:git-Paper-1059]
	... 19 more
[16:11:07] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to ItemJoin v3.9.1
org.bukkit.event.EventException
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:46) ~[patched_1.11.2.jar:git-Paper-1059]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:233) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:200) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:196) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:946) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:26) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:5) ~[patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.11.2.jar:git-Paper-1059]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_121]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_121]
	at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:830) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:403) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:754) [patched_1.11.2.jar:git-Paper-1059]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:652) [patched_1.11.2.jar:git-Paper-1059]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: java.lang.IllegalStateException: Start block missed in BlockIterator
	at org.bukkit.util.BlockIterator.<init>(BlockIterator.java:170) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.util.BlockIterator.<init>(BlockIterator.java:233) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.util.BlockIterator.<init>(BlockIterator.java:268) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.entity.CraftLivingEntity.getLineOfSight(CraftLivingEntity.java:178) ~[patched_1.11.2.jar:git-Paper-1059]
	at org.bukkit.craftbukkit.v1_11_R1.entity.CraftLivingEntity.getLineOfSight(CraftLivingEntity.java:204) ~[patched_1.11.2.jar:git-Paper-1059]
	at me.RockinChaos.itemjoin.Listeners.InteractCmds.getNearestEntityInSight(InteractCmds.java:64) ~[?:?]
	at me.RockinChaos.itemjoin.Listeners.InteractCmds.onInteractCmds(InteractCmds.java:50) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor129.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:44) ~[patched_1.11.2.jar:git-Paper-1059]
	... 19 more
```

**Comment by RockinChaos** [comment]
Can you provide a items.yml & config.yml.
Also do you know how this error occured? A certain action trigger it?

**Comment by mibby** [comment]
Items.yml - http://paste.ubuntu.com/24050455/
Config.yml - http://paste.ubuntu.com/24050457/

I'm not sure how this error occurred, just noticed it in the logs when searching for any exceptions today. I recently added a new world but did not attach it to ItemJoin. Perhaps the exception occurred when someone was put in the world or taken out of it?

**Comment by RockinChaos** [comment]
hmm It looks like its almost bound to a command. I will take this and test it. If you happen to realize what action is causing the error let me know and Ill bug test that action first.

**Comment by RockinChaos** [comment]
Hey i have been looking into this for the next update that I am about to release, It would be really helpful if you could provide a items.yml & config.yml.

**Comment by RockinChaos** [comment]
Possibly fixed an issue with this in v3.9.2 but I have no idea until you get back to me on this download it and let me know if you still have issues.

**Comment by RockinChaos** [comment]
Update has been released, you can get it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=147427
